src/modules/CreateClassForm/components/CreateClassForm.js

```diff
@@ -23,13 +31,7 @@ function CreateClassForm({ onSubmit }) {
 
       <FormGroup>
         <Label>Time</Label>
-        <FormField
-          fieldId="time"
-          name="time"
-          options={[{ value: '1', label: 'Imagine times' }]}
-          type={Select} // TODO add time select component
-          placeholder="Add class time"
-        />
+        <TimeInput />
```

Comment:
Did using a `FormField` with `type={TimeInput}` not work? FormField gives us the user-entered data onChange so we can display it in the preview, etc.

Reply:
It would probably work if I tried it :)